app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,89 @@
+body {
```

Comment:
i am expecting this file to organize webpage content's formate and customize things, i.e: generate *red* error messages. When I comment out this file, I don't see any difference either, which is odd. I will figure it out tmr. But we do have the option not to have this file generated.  `rails g scaffold MyModel --no-stylesheets`